packages/jaeger-ui/src/utils/readJsonFile.test.js

```diff
@@ -72,4 +72,17 @@ describe('fileReader.readJsonFile', () => {
     const p = readJsonFile({ file });
     return expect(p).rejects.toMatchObject(expect.any(Error));
   });
+
+  it('loads multiple JSON data', () => {
+    let fileContent;
+    try {
+      fileContent = fs.readFileSync('src/utils/fixtures/otlp2jaeger-multi-in.json.txt', 'utf-8');
+    } catch (err) {
+      console.error('Error reading file:', err);
+      throw err;
+    }
+    const file = new File([fileContent], 'multi.json', { type: 'application/json' });
+    const p = readJsonFile({ file });
+    expect(p).resolves.toBeDefined();
```

Comment:
is it possible to have somewhat stronger validation? Like checking the trace ID and # of spans,

Reply:
I'll look into this

Reply:
I'm not sure what can be done here. I'll work on this at a later point of time, lets leave it as such for now?